src/Codegen/Builders/Fields/IntrospectionSchemaFieldBuilder.hack

```diff
@@ -0,0 +1,27 @@
+namespace Slack\GraphQL\Codegen;
+use type Facebook\HackCodegen\{HackBuilder};
+
+final class IntrospectSchemaFieldBuilder extends FieldBuilder {
+    const type TField = shape(
+        'name' => string,
+        'output_type' => shape('type' => string, ?'needs_await' => bool),
+        ...
+    );
+
+    public function __construct() {
+        parent::__construct(shape(
+            'name' => '__schema',
+            'output_type' => shape('type' => '__Schema::nullableOutput()'),
```

Comment:
It would be nice at some point not to need hardcoded strings here, in case we ever want to change these method names.

Reply:
This is a little annoying because `__Schema` is something that gets generated by introspection so we can't use `output_type` which uses reflection

Reply:
Maybe we could refactor `output_type` to not rely on introspection, just as we did for FieldBuilder?

Anyway, not a priority.

Reply:
Maybe! This seems like such a specific thing though that I'd rather do this for now 😄